code/clue.js

```diff
@@ -30,35 +114,132 @@ const suspects = [
   mrMustard
 ]
 
-const weapons = []
+const weapons = [
+  rope,
+  knife,
+  candlestick,
+  dumbbell,
+  poison,
+  axe,
+  bat,
+  trophy,
+  pistol
+]
+
+const rooms = [
+  "dining room",
+  "conservatory",
+  "kitchen",
+  "study",
+  "library",
+  "billiard room",
+  "lounge",
+  "ballroom",
+  "hall",
+  "spa",
+  "living room",
+  "observatory",
+  "theater",
+  "guestHouse",
+  "patio"
+]
 
-const rooms = []
+//IS EVERYTHING THERE?
+console.log(suspects, weapons, rooms)
 
 // THIS FUNCTION WILL RANDOMLY SELECT ONE ITEM FOR THE ARRAY THAT YOU PASS IN TO THE FUNCTION.
+
 const randomSelector = array => {
   return array[Math.floor(Math.random() * array.length)]
 }
 
+
 // CREATE AN OBJECT THAT KEEPS THE MYSTERY.
 // With a killer, a weapon and a room.
 // The values will be set later.
 
+const mystery = {
+  killer: "",
+  weapon: "",
+  room: ""
+}
+
+console.log(mystery)
+
 // FINISH THIS FUNCTION TO SHOW ALL INFORMATION ABOUT THE KILLER.
 // This function will be invoked when you click on the killer card.
+//PICKKILLER
 const pickKiller = () => {
   // This will randomly select a killer from the suspects. And add that to the mystery object.
   mystery.killer = randomSelector(suspects)
 
   // This will change the background color of the card to the one connected to the chosen killer and show the full name of the killer.
   const theKiller = document.getElementById("killer")
   const theKillerName = document.getElementById("killerName")
+  const theKillerAge = document.getElementById("killerAge")
+  const theKillerOccupation = document.getElementById("killerOccupation")
+  const theKillerImage = document.getElementById("killerImage")
+  const theKillerDescripton = document.getElementById("killerDescription")
 
   theKiller.style.background = mystery.killer.color
   theKillerName.innerHTML =
     mystery.killer.firstName + " " + mystery.killer.lastName
+  theKillerAge.innerHTML = mystery.killer.age
+  theKillerOccupation.innerHTML = mystery.killer.occupation
+  theKillerImage.src = mystery.killer.image
+  theKillerDescripton = ""
```

Comment:
Here you get a type error! Why? It's because you have declared the `theKillerDescription` variabel with a `const`. Here you can read about what it means: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Errors/Invalid_const_assignment